lib/cloudex.ex

```diff
@@ -21,15 +22,15 @@ defmodule Cloudex do
     Uploads a (list of) image file(s) and/or url(s) to cloudinary
   """
   @spec upload(list | String.t()) :: upload_result
-  @spec upload(list | [String.t()], map) :: upload_result
-  def upload(list, options \\ %{}) do
+  @spec upload(list | [String.t()], map, integer()) :: upload_result
+  def upload(list, options \\ %{}, recv_timeout \\ @default_recv_timeout) do
```

Comment:
instead of passing in only the timeout, I think it might be beneficial to have a keyword list argument, so it would be possible to pass in other httpoison options(https://hexdocs.pm/httpoison/HTTPoison.Request.html). so the input would be something like [recv_timeout: 60_0000, max_redirect: 3, proxy_auth: {user, "password}]